stor/base.py

```diff
@@ -86,8 +87,126 @@ def _has_incompatible_path_module(self, other):
         compatible path module for path operations"""
         return isinstance(other, Path) and other.path_module != self.path_module
 
-    copy = utils.copy
-    copytree = utils.copytree
+    def copy(self, dest, **swift_retry_options):
```

Comment:
I think this method should be on FileSystemPath, not the base path class.

Also let's get rid of swift_retry_options :)

Reply:
I moved it to FileSystemPath. Are we sure we want to get rid of swift_retry_options? It's a big api break. And will have changes both upstream (packages that invoke `stor.copy(..., ..., swift_retry_options=...)` and downstream (upload, download_object, download all have declarations like `def upload(self, to_upload, **kwargs):`, also tests will change). Let me know.